src/main/java/nextstep/domain/Line.java

```diff
@@ -21,25 +25,16 @@ public Line(List<Boolean> points) {
         this.points = points;
     }
 
-    public static boolean validateHasOneOrMoreTransverseBar(List<Boolean> line) {
-        return line.stream()
-                .anyMatch(Boolean::booleanValue);
-    }
-
-    public static boolean validateNotOverlap(List<Boolean> line) {
-        return IntStream.range(0, line.size() - 1)
-                .mapToObj(i -> new LineValidateOverlapDto(line.get(i), line.get(i + 1)).isOverlap())
+    private boolean validateNotOverlap(List<Boolean> points) {
+        return IntStream.range(0, points.size() - 1)
+                .mapToObj(i -> new LineValidateOverlapDto(points.get(i), points.get(i + 1)).isOverlap())
                 .noneMatch(Boolean::booleanValue);
     }
 
     public List<Boolean> getPoints() {
         return points;
```

Comment:
Collections.unmodifiablelist()를 활용하여 read-only로 리턴하여 외부에서 컬렉션의 변조를 막도록 하면 좋을 것 같아요 :)
```suggestion
        return Collections.unmodifiablelist(points);
```

Reply:
앗.. 깜빡했습니다..ㅠㅠ